### PR TITLE
issue 163 - pr/add advisor uni info

### PR DESCRIPTION
**Pull request by antonleviathan**
Closes https://github.com/statebox/pm/issues/163
I referred to 152 in the commit by accident!

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ plugins:
   - jekyll-compress-images
 
 compress_images:
-  images_path: "assets/images/**/*.{gif,png,jpg,jpeg,svg}"
+  images_path: "assets/images/**/*.{gif,png,jpg,jpeg}"
```
Comment (wires) [inline _config.yml]: why is svg removed? also if we have two extensions, jpg and jpeg, we might want to move those all to the same extension

File: _includes/list.html [hunk 2]
```diff
@@ -1,5 +1,9 @@
 <ul>
 {% for item in include.items %}
-  <li><a href="/{{ include.collection }}/{{ item.file }}">{{ item.fullname }}</a></li>
+  {% if item.member == false %}
```
Comment (wires) [inline _includes/list.html]: you should move this if statement into the `<li>`

File: _people/bob.md [hunk 3]
```diff
@@ -5,6 +5,7 @@ name: Bob
 file: bob
 fullname: Bob Coecke
 location: Oxford, England
+position: Oxford University Quantum Group
```
Comment (wires) [inline _people/bob.md]: this should probably be called `affiliation`

File: _includes/list.html (hunk 2 above)
Comment (wires) [reply _includes/list.html]: in fact, it should probably be `{% if item.affiliation ... %}` etc

File: _people/jelle.md [hunk 4]
```diff
@@ -5,6 +5,7 @@ name: Jelle
 file: jelle
 fullname: Jelle Herold
 location: Amsterdam, NL
+position:
```
Comment (wires) [inline _people/jelle.md]: don't add it if it's not filled it, less is more!

**Comment by wires** [comment]
I don't like the way this looks,

screen shot 2018-03-21 at 18 02 33

instead of `Name | Faculty`, probably only the name should be underlined, and maybe `–` (em-dash) as separator and/or making the affiliation italic

**Comment by wires** [comment]
also referring to the issue in the commit must be of the form `closes statebox/pm#1234`

File: _config.yml (hunk 1 above)
Comment (antonleviathan) [reply _config.yml]: svgo worker: `svgo` not found; please provide proper binary or disable this worker (--no-svgo argument or `:svgo => false` through options)

This error pops up.

**Comment by wires** [comment]
screen shot 2018-03-21 at 18 11 48

this looks better

File: _config.yml (hunk 1 above)
Comment (wires) [reply _config.yml]: okay, thanks.

So probably this `svgo` is a svg optimizer that was added when you added the image optimization stuff.

As it requires some binary this is not going to work, unless there is some gem that provides this. It's also arguable if we need this optimization or not, I don't know how many `.svg` we have and what the result is of running the optimizer; so I'm okay with removing this

**Comment by antonleviathan** [comment]
Cleaned up the list.html include as you recommended.

**Comment by FabrizioRomanoGenovese** [comment]
The way things are presented now is inconsistent. For some people only the affiliation is listed, for other people also their role.

    Bob Coecke - Oxford University Quantum Group
    Brendan Fong - MIT Department of Mathematics
    David I. Spivak - MIT Department of Mathematics
    Fredrik Nordvall Forsberg - Stratchlyde Mathematically Structured Programming Group
    Neil Ghani - Head of Stratchlyde Mathematically Structured Programming Group
    Pawel Sobocinski - Faculty Member - School of Electronics and Computer, University of Southampton
    Philip Zahn - Assistant Professor - University of St. Gallen
    Viktor Winschel - PhD Economics, University of Mannheim

Please restate as


    Bob Coecke - Oxford University Quantum Group
    Brendan Fong - MIT Department of Mathematics
    David I. Spivak - MIT Department of Mathematics
    Fredrik Nordvall Forsberg - Stratchlyde Mathematically Structured Programming Group
    Neil Ghani - Head of Stratchlyde Mathematically Structured Programming Group
    Pawel Sobocinski - School of Electronics and Computer, University of Southampton
    Philip Zahn - University of St. Gallen
    Viktor Winschel - OICOS   <-- Vik is not in uni anymore, so we should list him as affiliated to OICOS.

**Pull request merged**